frontend/src/metabase/parameters/utils/dashboard-options.js

```diff
@@ -56,19 +56,19 @@ export function getDefaultOptionForParameterSection() {
       }
 
       if (sectionId === "location") {
-        defaultOption = options.find(o => o.name === "Is");
+        defaultOption = options.find(o => o.type === "string/=");
```

Comment:
we need to support other languages and thanks god we have tests for it

Reply:
Oh wow :man_facepalming: